crucible-jvm/src/Lang/Crucible/JVM/Simulate.hs

```diff
@@ -625,7 +623,7 @@ setupCrucibleJVMCrux cb verbosity sym p cname mname args = do
      (JVMHandleInfo _ h) <- findMethodHandle ctx mcls meth
 
 
-     let failIfNotEqual :: forall f m a (b :: k).
+     let failIfNotEqual :: forall k f m a (b :: k).
```

Comment:
What is the warning here?  Pretty sure GHC 8.10 is going to require this quantification over `k`

Reply:
I don't remember the error message exactly, but I think it said something about how the explicit quantification would be required in the future.

Reply:
oops, I misread the patch, I thought it was removing the quantification over `k`